simulation/traffic_simulator/src/entity/entity_base.cpp

```diff
@@ -217,6 +217,37 @@ auto EntityBase::getLaneletPose(double matching_distance) const
   }
 }
 
+auto EntityBase::refillEntityStatusWithLaneletData(
+  traffic_simulator_msgs::msg::EntityStatus & status) const -> void
+{
+  const auto unique_route_lanelets = traffic_simulator::helper::getUniqueValues(getRouteLanelets());
+
+  std::optional<traffic_simulator_msgs::msg::LaneletPose> lanelet_pose;
+
+  if (unique_route_lanelets.empty()) {
+    lanelet_pose =
+      hdmap_utils_ptr_->toLaneletPose(status.pose, getStatus().bounding_box, false, 1.0);
+  } else {
+    lanelet_pose = hdmap_utils_ptr_->toLaneletPose(status.pose, unique_route_lanelets, 1.0);
+    if (!lanelet_pose) {
+      lanelet_pose =
+        hdmap_utils_ptr_->toLaneletPose(status.pose, getStatus().bounding_box, false, 1.0);
```

Comment:
```
hdmap_utils_ptr_->toLaneletPose(status.pose, getStatus().bounding_box, false, 1.0);
```
3 rd argument of this function means include crosswalk lanelet or not.
For pedestrian entity, it should be true.

Reply:
@lchojnack 
The comment above meant that 
[the 3rd argument ](https://github.com/tier4/scenario_simulator_v2/blob/24617d21fb0a221eaf67a6bee7a53d0eeebff69a/simulation/traffic_simulator/include/traffic_simulator/hdmap_utils/hdmap_utils.hpp#L75C59-L75C72)of the function " toLaneletPose " can be true or false according to the kind of Entity. 
For the pedestrian entity, the argument should be true, for the other entities the argument is supposed to be false.
Could you take this specification into consideration ?